config/reports.yaml

```diff
@@ -28,5 +28,14 @@ htmlVolume:
 
 replicaCount: 2
 
+# Specify the "hard" scheduling constraints
 nodeSelector:
-  agentpool: x86medium
+  # Ensure ARM64 is used to decrease cloud bill (instead of using `amd64`
+  kubernetes.io/arch: arm64
+
+# Tolerates to run on tainted `arm64` nodes
```

Comment:
```suggestion
```

Reply:
This comment only repeats what is written below and does not explain the "why": better remove